### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by kiwikern**
zoom level and map center are static

### CODE REVIEW

**Comment by kiwikern** [comment]
I did not commit the rebuilt dist folder

**Review by BenjaminKaul — APPROVED** [review]

**Review by o3q3 — COMMENTED** [review]
bos3iid7@gmail.com

**Review by o3q3 — CHANGES_REQUESTED** [review]
Pl

**Review by o3q3 — COMMENTED** [review]
[saad](Instagram.com/5)

File: public/src/main.js [hunk 1]
```diff
@@ -41,6 +43,14 @@ const firebaseConfig = {
 
   firebase.initializeApp(firebaseConfig);
 
+import { Icon } from 'leaflet';
+
+delete Icon.Default.prototype._getIconUrl;
+Icon.Default.mergeOptions({
```
Comment (o3q3) [inline public/src/main.js:49]: bos3iid7@gmail.com

**Review by o3q3 — COMMENTED** [review]
@7p

**Comment by o3q3** [comment]
thefloyd77@gmail.com

**Pull request merged**